types/prop-types/index.d.ts

```diff
@@ -27,9 +39,8 @@ export interface Requireable<T> extends Validator<T | undefined | null> {
 export type ValidationMap<T> = { [K in keyof T]-?: Validator<T[K]> };
```

Comment:
Unrelated, but I really hate the `-?` here. It forces every component to define a prop type for every interface property, even for those inherited/extended. This is entirely overkill.